Flutter/lib/AppTheme.dart

```diff
@@ -0,0 +1,91 @@
+import 'import_all.dart';
```

Comment:
Why would we need to import everything? Seems excessive

Reply:
Every time when you do `package:flutter/material.dart`, you're importing all dependencies needed for your class. If you have a look at `material.dart` [repo](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/material.dart) that was written by the Flutter team, you'll notice they're exporting all dependencies you might use. So, I followed the convention the Flutter team uses. It's my understanding that Flutter does a pretty good job at managing dependencies you won't use. If you still want to change it, let me know.

Reply:
Resolved in commit `app-style 89939db`

Reply:
Ah, I see what you mean now. What the Flutter team did is what seems to be called a library in Dart (or a package in Java, like `java.util`) - notice line 17 `library material` and how every export is `src/material/*.dart`. The purpose here is so developers using Flutter only `import package:flutter/material.dart` and all _related_ files will be imported. The equivalent for us would be something like `import package:settle/settle.dart` that will import files like Settle.dart and other files a developer might need to interact with our code (so the http package should be excluded because we handle that in Settle.dart). Our codebase is not yet large enough to need a library, but we might get there one day 🙂